patch.py

```diff
@@ -39,7 +39,7 @@ def update_change_list(message):
 
 currentVersion = get_ripme_json()["latestVersion"]
 
-print('Current version ' + currentVersion)
+print(f'Current version {currentVersion}')
```

Comment:
> Lines `42-80` refactored with the following changes:
> 
>     * Use f-string instead of string concatenation [×15] ([`use-fstring-for-concatenation`](https://docs.sourcery.ai/Reference/Default-Rules/refactorings/use-fstring-for-concatenation/))
> 
>     * Replace call to format with f-string [×2] ([`use-fstring-for-formatting`](https://docs.sourcery.ai/Reference/Default-Rules/refactorings/use-fstring-for-formatting/))